store/querier.go

```diff
@@ -2,6 +2,7 @@ package store
 
```

Comment:
I think this file can be reorged a bit. cc/ @hamdiallam , I was think we put in a new directory query/ within store/. Then have a file for routing queries, handling output/deposit queriers and a file for block queriers. Not sure if hamdi has started this change, but since the majority of this file is changed in this pr, I think he should leave it how it is in his and then changes should be made here

Reply:
Unless its necessary and would clearly simplify this current PR, let's do this later